FireboltNETSDK/Client/FireboltConnection.cs

```diff
@@ -137,6 +134,10 @@ internal int InfraVersion
         /// <returns>The state of the connection.</returns>
         public override ConnectionState State => _connectionState.State;
 
+        /// <summary>
+        /// Gets database version.
+        /// </summary>
+        /// <returns>The version of the database.</returns>
         public override string ServerVersion
```

Comment:
Summary and returns tags are somewhat redundant here (and in some other places).
Can one of them be omitted? What's the best practice here?

Reply:
I really do not know about the best practice. You are right that this may look redundant but I saw the same both in java and in C#.